src/osd/OSD.cc

```diff
@@ -5317,7 +5317,7 @@ PGRef OSD::handle_pg_create_info(const OSDMapRef& osdmap,
   }
 
   pg->handle_initialize(rctx);
```

Comment:
I think I already approved the PR with this commit?  Rebase this PR on that one.